hawc/apps/assessment/api.py

```diff
@@ -170,6 +178,61 @@ def public(self, request):
         serializer = serializers.AssessmentSerializer(queryset, many=True)
         return Response(serializer.data)
 
+    @method_decorator(cache_page(60 * 60 * 24))
+    @action(detail=False, methods=("get",), renderer_classes=PandasRenderers)
+    def training_data(self, request):
+        Endpoint = apps.get_model("animal", "Endpoint")
+        # map of django field names to friendlier column names
+        column_map = {
+            "assessment_id": "assessment_uuid",
+            "animal_group__species__name": "species_name",
+            "animal_group__strain__name": "strain_name",
+            "system": None,
```

Comment:
almost forgot; can we also add "organ", "effect", "effect_subtype", "data_location",  "name", "animal_group__sex",  "animal_group__experiment__chemical",